packages/github/src/schemas/pull_requests.mjs

```diff
@@ -1,4 +1,4 @@
-import * as z from "zod";
+import { z } from "zod";
```

Comment:
it must've been some autoimport fix of sorts, I remember having `import { z } `.

Tks for fixing it!